config/default.yaml

```diff
@@ -295,6 +295,11 @@ tracker:
   private: true
   # Reject peers that do a lot of announces (could improve privacy of TCP/UDP peers)
   reject_too_many_announces: false
+  # This is a comma-separated list of additional trackers
+  # will be used in creation of webtorrents if specified
+  # example: 
+  #webtorrent_trackers_list_csv: 'udp://tracker.leechers-paradise.org:6969,udp://tracker.coppersurfer.tk:6969,wss://tracker.btorrent.xyz'
+  webtorrent_trackers_list_csv: ''
```

Comment:
Use an array and a root key instead:

```
tracker:
  enabled: true
  ...

additional-trackers:
  urls:
    - 'wss://tracker1'
    -  ...
```

Reply:
@Chocobozzz 
I need help, I could not make this work with my server.
this happens: 
![Selection_181](https://user-images.githubusercontent.com/4972818/214012143-305249d0-a93d-48e8-9277-79748f4882b5.png)

Looks like config is being moderated from somewhere and I am not allowed to add things to config?
or is it just a yaml space problem?